.github/workflows/objective_c.yaml

```diff
@@ -0,0 +1,44 @@
+name: objective_c
```

Comment:
Maybe it is better to fold this into `ffigen.yml` as a matrix. That way we don't have to duplicate things.

> For testing, I want to avoid having the package:objective_c tests use ffigen. So there's not much I can test at the moment, since the only things in the package are internal details of ffigen. Those are all covered by the ffigen tests anyway. There will be more to test in this package once I add some of the higher level classes, like NSString.

Either we should figure out how to do coverage from the `package:ffigen` run, or we should have tests in this package. (@HosseinYousefi what's the situation for `packag:jni` in this regard?)

Reply:
JNI has more classes (like `JString`, `JList`, ...). And we also test `Jni.env` stuff. It makes sense to have a separate workflow for those. However, the coverage doesn't get added. I have an open issue for that: #1010.